tests/sentry/models/test_lostpasswordhash.py

```diff
@@ -28,3 +28,23 @@ def test_send_recover_mail(self):
             args=[password_hash.user_id, password_hash.hash],
         )
         assert url in msg.body
+
+    def test_send_relocation_mail(self):
+        password_hash = LostPasswordHash.objects.create(user=self.user)
+
+        request = HttpRequest()
+        request.method = "GET"
+        request.META["REMOTE_ADDR"] = "1.1.1.1"
+
+        with self.options({"system.url-prefix": "http://testserver"}), self.tasks():
+            LostPasswordHash.send_email(self.user, password_hash.hash, request, "relocate_account")
+
+        assert len(mail.outbox) == 1
```

Comment:
I think maybe worth including some more tests of the message body here, particularly that we mention the right ords?

Reply:
What do you mean by ords? I added a test to check more of the message body